sdk/src/main/java/io/radar/sdk/RadarVerifiedReceiver.kt

```diff
@@ -17,4 +17,12 @@ abstract class RadarVerifiedReceiver {
      */
     abstract fun onTokenUpdated(context: Context, token: RadarVerifiedLocationToken)
 
+    /**
+     * Tells the receiver that a verified request failed.
+     *
+     * @param[context] The context.
+     * @param[status] The status.
+     */
+    abstract fun onVerifiedError(context: Context, status: Radar.RadarStatus)
```

Comment:
Do we definitely want another receiver? Might be easiest / simpler to use the existing `onError` receiver right?

Reply:
I was thinking in particular re: errors from `getConfig` in `initialize` call, but it looks like we actually don't `sendError` from there. Maybe we'd want to do that?